cmd/substreams/auth.go

```diff
@@ -3,10 +3,11 @@ package main
 import (
 	"errors"
 	"fmt"
-	"github.com/streamingfast/cli"
 	"os"
 	"strings"
 
+	"github.com/streamingfast/cli"
+
 	"github.com/charmbracelet/huh"
```

Comment:
```suggestion
	"github.com/streamingfast/cli"
    "github.com/charmbracelet/huh"
```

And a another round of `goimports`.

Reply:
Thanks.